docs/content/topics/developers.md

```diff
@@ -262,6 +262,18 @@ $ helm install -n brigade brigade/brigade -f myvalues.yaml
 
 Don't forget to also create a project.  Check out [projects](./projects.md) to see how it's done.
 
+## Developing brigade with kind
+
+You can also use [kind](https://github.com/kubernetes-sigs/kind) for your day to day Brigade development workflow.
+
+- Run `kind create cluster` to create the cluster 
+- Run `export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"`
+- Run `helm init` to install helm on the `kind` cluster. The latest [Helm 3](https://github.com/helm/helm) version is recommended. However, if you're running Helm 2, check [here](https://helm.sh/docs/using_helm/#role-based-access-control) for proper Helm/Tiller installation instructions
```

Comment:
Thank you for providing the excuse to play around with helm 3!

Since I still require helm v2 to interact with other clusters, I placed helm v3 into `helm3` locally.  Is this a common use case?  If so, I wonder if it'd be worth a follow-up PR enabling switching out helm versions for the `make helm-install` command... something like:
```
# Allow for users to supply a different helm cli name,
# for instance, if one has helm v3 as `helm3` and helm v2 as `helm`
HELM ?= helm
...
helm-upgrade:
	$(HELM) upgrade --install $(BRIGADE_RELEASE) brigade/brigade --namespace $(BRIGADE_NAMESPACE) \
...
```

Reply:
Oh, this is on me and Ralph... we pointed Dimitris to Helm 3 for this.
Can we maybe have two targets? I really think it's valuable for people to get familiar with Helm 3, but still provide the Helm 2 target.
WDYT?

Reply:
I opted for @vdice's option, since it's easy to change the variable on the Makefile and by adding more targets we would have 4 in total: `helm2-install`, `helm2-upgrade`, `helm3-install`, `helm3-upgrade`.